deploy/clowdapp.yaml

```diff
@@ -414,6 +419,12 @@ parameters:
   - description: Postgres driver logging level (trace, debug, info, warn, error, fatal, panic)
     name: DATABASE_LOGGING_LEVEL
     value: "debug"
+  - description: RBAC checking enabled
+    name: APP_RBAC_ENABLED
+    value: "false"
```

Comment:
I am proposing to re-review this patch and merge it as-is with RBAC currently disabled so tests will pass and everything will operate normally.

I will file a new one-liner PR that enables RBAC which will fail tests so @akhil-jha can work on tests.

Reply:
I love the approach, hope @akhil-jha is ok with that :)